winlogbeat/docs/getting-started.asciidoc

```diff
@@ -150,6 +150,26 @@ Before starting {beatname_uc}, modify the user credentials in
 +{beatname_lc}.yml+ and specify a user who is
 <<privileges-to-publish-events,authorized to publish events>>.
 
+By default, the `winlogbeat` service runs as `Local System` account. If you want to run `winlogbeat` service as a local user account which is not an Administrator, follow the below steps. By following these steps, your local user account will be granted `Log on as a service` security policy and made part of group `Event Log Readers` to read the event log. (You can skip these steps if you want to continue running Winlogbeat as an Administrator.)
+
+. Open the Services Management console in Windows, if not already open by running this command:
```

Comment:
```suggestion
. Open the Services Management console with this command:
```

"on Windows" is implicit for Winlogbeat

"if not already open" - that seems unnecessary.

Reply:
updated as suggested